p2p/p2p.go

```diff
@@ -186,7 +187,8 @@ func (s *Service) Ctx() context.Context {
 
 func (sc *Config) buildOpts() ([]libp2p.Option, error) {
 	// TODO: get external ip
-	ip := "0.0.0.0"
+	// ip := "0.0.0.0"
+	ip := "127.0.0.1"
```

Comment:
why is this changed? 0.0.0.0 allows for the ip to be replaced with the actual ip